tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -81,8 +81,9 @@ def test_multiple_jobs_submission(scheduler, region, pcluster_config_reader, clu
 
 @pytest.mark.regions(["sa-east-1"])
 @pytest.mark.instances(["c5.xlarge"])
-@pytest.mark.schedulers(["sge", "torque"])
-@pytest.mark.usefixtures("region", "os", "instance")
+@pytest.mark.schedulers(["sge"])
```

Comment:
let's leave torque here as well since the logic is slightly different

Reply:
ok